joomla/joomla-cms
# [3.7.x] Input field invalid highlight

Pull Request for Issue #12438  .
### Summary of Changes

Adds highlight (border & text) to input fields if invalid.
### Testing Instructions

Navigate to Global Configuration -> System. Check 'Session Lifetime' field with non integer number (0.5).

**Before Patch**
![session-field1](https://cloud.githubusercontent.com/assets/2803503/19419840/bf772dca-93d7-11e6-9477-e5f928de262e.png)

**After Patch**
![session-field2](https://cloud.githubusercontent.com/assets/2803503/19419843/c2e974b8-93d7-11e6-82b5-75c8477ebe60.png)
### Documentation Changes Required

None
